ansible/roles/test/files/ptftests/hash_test.py

```diff
@@ -52,13 +52,9 @@ def setUp(self):
 
         self.src_ip_range = [unicode(x) for x in self.test_params['src_ip_range'].split(',')]
         self.dst_ip_range = [unicode(x) for x in self.test_params['dst_ip_range'].split(',')]
-        self.test_hash_srcip = self.test_params.get('hash_srcip', True)
-        self.test_hash_dstip = self.test_params.get('hash_dstip', True)
-        self.test_hash_srcport = self.test_params.get('hash_srcport', True)
-        self.test_hash_dstport = self.test_params.get('hash_dstport', True)
-        self.test_hash_inport = self.test_params.get('hash_inport', False)
-        self.test_ipv4 = self.test_params.get('ipv4', True)
-        self.test_ipv6 = self.test_params.get('ipv6', True)
+        self.src_ip_interval = lpm.LpmDict.IpInterval(ip_address(self.src_ip_range[0]), ip_address(self.src_ip_range[1]))
+        self.dst_ip_interval = lpm.LpmDict.IpInterval(ip_address(self.dst_ip_range[0]), ip_address(self.dst_ip_range[1]))
+        self.hash_keys = self.test_params.get('hash_keys', ['src-ip'])
```

Comment:
in original test, by default, we test srcip, dstip, srcport, dstport, can you keep it the same?

Reply:
> in original test, by default, we test srcip, dstip, srcport, dstport, can you keep it the same?

done.